### ISSUE TITLE
Images flicker when swiping

### ISSUE THREAD

**Issue by c0m1t** [reactrondev/react-native-web-swiper#67]
I have a simple `swiper` with couple of images inside. Everything is fine on web but in iOS (I have not tested android yet) I'm experiencing some unusual behavior. When I swipe on the emulator, I see the next image for a fraction of second and then it is gone, until the transition is finished.

```tsx
<View style={styles.swiperContainer}>
        <Swiper>
          <View>
            <Image
              source={{ uri: 'https://picsum.photos/seed/hero1/1080/720' }}
              style={{ height: '100%', width: '100%' }}
            />
          </View>
          <View>
            <Image
              source={{ uri: 'https://picsum.photos/seed/hero2/1080/720' }}
              style={{ height: '100%', width: '100%' }}
            />
          </View>
          <View>
            <Image
              source={{ uri: 'https://picsum.photos/seed/hero3/1080/720' }}
              style={{ height: '100%', width: '100%' }}
            />
          </View>
        </Swiper>
</View>

```
Is there any workaround/solution for this issue?

It might be related to #65, I'm not sure.

**Comment by jarredt** [comment]
Yeah, I think it's probably the same issue as #65 .

**Comment by jarvisluong** [comment]
This issue is solved in v2.2.2
